packages/components/modal/src/modal.tsx

```diff
@@ -0,0 +1,79 @@
+import { Heading } from "@buoysoftware/anchor-typography";
+import { Box } from "@buoysoftware/anchor-layout";
+import ReactModal, { Props as ReactModalProps } from "react-modal";
+import { rgba } from "polished";
+import { useTheme } from "@buoysoftware/anchor-theme";
+import { get } from "styled-system";
+
+ReactModal.setAppElement("#root");
+
+interface OwnProps {
+  bg?: string;
+  borderRadius?: string;
+  children?: React.ReactNode;
+  modalActions?: React.ReactNode;
+  testId: string;
+  title: string;
+  width?: string;
+}
+
+export type ModalProps = ReactModalProps & OwnProps;
+
+export const Modal: React.FC<ModalProps> = ({
+  bg = "white",
+  borderRadius = "modal",
+  children,
+  modalActions,
+  testId,
+  title,
+  width = "modalWidth",
+  ...modalProps
+}): React.ReactElement => {
+  const { colors, radii, sizes, space } = useTheme();
+
+  const contentWidth = get(sizes, width, width);
+  const contentRadius = get(radii, borderRadius, borderRadius);
+  const contentBackground = get(colors, bg, bg);
+
+  return (
+    <ReactModal
+      style={{
+        overlay: {
+          backgroundColor: rgba(colors.blue100, 0.64),
+          display: "flex",
+          alignItems: "flex-start",
+          justifyContent: "center",
+          zIndex: 99,
+        },
+        content: {
+          backgroundColor: contentBackground,
+          border: "none",
+          borderRadius: contentRadius,
+          bottom: undefined,
+          left: undefined,
+          marginTop: space.modal.gutterT,
+          padding: 0,
+          position: undefined,
+          right: undefined,
+          top: undefined,
```

Comment:
Are these styles a sort of override of some default? If not can they be dropped?

Reply:
Gonna go ahead and say yeah, they're overrides 😂 

<img width="959" alt="Screenshot 2023-03-15 at 5 01 11 PM" src="https://user-images.githubusercontent.com/3165407/225441471-fa50ec68-4046-4b2a-9dd7-dac1fcfc7186.png">

Reply:
Good call, though - I missed some spacing updates that were needed to bring this in line with the new theme. Those have also been updated!